### PR TITLE
Proposals reviews count

### PR DESCRIPTION
**Pull request by danielSanchezQ**

### CODE REVIEW

File: vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql [hunk 1]
```diff
@@ -95,6 +95,7 @@ CREATE VIEW full_proposals_info
 AS
 SELECT
     proposals.*,
+    coalesce(reviews_count, 0) as reviews_count,
```
Comment (mzabaluev) [inline vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql:98]: `coalesce` made me look into the docs for the semantics.
```suggestion
    ifnull(reviews_count, 0) as reviews_count,
```

File: vit-servicing-station-lib/src/v0/endpoints/proposals/handlers.rs [hunk 2]
```diff
@@ -33,11 +34,17 @@ pub mod test {
...
+        let mut proposal: FullProposalInfo = proposals_testing::get_test_proposal();
         proposals_testing::populate_db_with_proposal(&proposal, &pool);
         let challenge: Challenge =
             challenges_testing::get_test_challenge_with_fund_id(proposal.proposal.fund_id);
         challenges_testing::populate_db_with_challenge(&challenge, &pool);
+
+        let review = reviews_testing::get_test_advisor_review_with_proposal_id(
+            proposal.proposal.proposal_id.parse().unwrap(),
+        );
+        reviews_testing::pupulate_db_with_advisor_review(&review, &pool);
```
Comment (mzabaluev) [inline vit-servicing-station-lib/src/v0/endpoints/proposals/handlers.rs:46]: Typo: populate

**Review by mzabaluev — APPROVED** [review]
Good work!

File: vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql (hunk 1 above)
Comment (danielSanchezQ) [reply vit-servicing-station-lib/migrations/2020-05-22-112032_setup_db/up.sql:98]: Now that I check on it sqlite support `ifnull`, but for some reason in the docs I first checked it forward me to `coalesce`

**Pull request merged**